server/src-lib/Hasura/Server/Telemetry.hs

```diff
@@ -128,8 +128,8 @@ runTelemetry (Logger logger) manager cacheRef dbId instanceId = do
 
 computeMetrics :: SchemaCache -> Metrics
 computeMetrics sc =
-  let nTables = Map.size $ Map.filter (isNothing . tiViewInfo) usrTbls
-      nViews  = Map.size $ Map.filter (isJust . tiViewInfo) usrTbls
+  let nTables = Map.size $ Map.filter (isNothing . _tiViewInfo) usrTbls
```

Comment:
Maybe we can track the number of enum tables being used?

Reply:
Added to the metrics.